src/app/tech-carbon-estimator/tech-carbon-estimator.component.spec.ts

```diff
@@ -103,42 +103,15 @@ describe('TechCarbonEstimatorComponent', () => {
     expect(component.showEstimation).toBeFalse();
   });
 
-  it('should scroll to top of assumptions and limitation when showAssumptionsAndLimitation is called', () => {
-    component.showEstimation = true;
-    component.showAssumptionsAndLimitationView = true;
-    fixture.detectChanges();
-    spyOn(component.assumptionsLimitation.nativeElement, 'scrollIntoView').and.callThrough();
-    component.showAssumptionsAndLimitation();
-    fixture.detectChanges();
-
-    expect(component.assumptionsLimitation.nativeElement.scrollIntoView).toHaveBeenCalled();
-  });
-
-  it('should scroll to top of estimations when closeAssumptionsAndLimitation is called', () => {
```

Comment:
The equivalent test would now be to check that the open button is scrolled into view when this is called but the open button is only on the page when the assumptions and limitations view is closed it can't be spied on before the close button is pressed.

Reply:
I guess if we wanted to keep this test we could hide the button rather than make it non-existent. Or scroll to something that does always exist, though the behaviour of this is starting to feel a bit complicated and I wouldn't want to add more.
To be honest, I'm not sure if the scroll is ever needed, as you end up in the right place if the button is focussed. The only time it doesn't try to focus is if you hit escape while focussed outside of the assumptions - which means you've either clicked off it or tabbed all the way through to the diagram. Do we really care where the form is scrolled to in that case? Or it could be scroll is the alternative to focus rather than allowing both separately.

Reply:
Yes, it seems the explicit scrolling is not required. I've removed it so it just has the focus behaviour.